### ISSUE TITLE
Ubuntu 18.04 LTS: dnsmasq is not a service

### ISSUE THREAD

**Issue by DL6ER** [pi-hole/pi-hole#2202]
**In raising this issue, I confirm the following:**

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] The issue I am reporting can be *replicated*.
- [X] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the the source code relevant to this issue?:**

## 10

---
**Expected behaviour:**

No error is printed on installing Pi-hole on Ubuntu 18.04 LTS after #2180 has been merged

**Actual behaviour:**

![screenshot at 2018-05-31 13-52-51](https://user-images.githubusercontent.com/16748619/40780801-f9e42e20-64d9-11e8-80f6-64b07f0dc5bd.png)

**Steps to reproduce:**

Install Pi-hole/FTL/AdminLTE `development` + ( #2180 ) on Ubuntu 18.04 LTS:

```
curl dl6er.de/pi.hole | bash
mkdir /etc/pihole
echo "FTLDNS" | sudo tee /etc/pihole/ftlbranch
```

**Problem:***
https://github.com/pi-hole/pi-hole/blob/8523c0da32e8eadaa78f26a8d9194c0c4ba2f93f/automated%20install/basic-install.sh#L1961-L1967
https://github.com/pi-hole/pi-hole/blob/8523c0da32e8eadaa78f26a8d9194c0c4ba2f93f/automated%20install/basic-install.sh#L2081-L2085

`if which dnsmasq` is unsuitable to detect if the binary is also available as service, see:
```
root@ubuntu-s-2vcpu-2gb-fra1-01:/etc/.pihole# which dnsmasq
/usr/sbin/dnsmasq
root@ubuntu-s-2vcpu-2gb-fra1-01:/etc/.pihole# systemctl status dnsmasq
Unit dnsmasq.service could not be found.
```

**Comment by DL6ER** [comment]
@PromoFaux You contributed this code in https://github.com/pi-hole/pi-hole/commit/76654c7856b00a25ace1dd31019f78337d59f5a0 Do you also want to change it?

**Comment by PromoFaux** [comment]
Huh, I could have sworn I'd worked around this... I'll take a look later

**Comment by RamSet** [comment]
I addressed a similar issue under Ubuntu 16.0 for dnsmasq via Network Manager under [https://github.com/pi-hole/pi-hole/pull/2218]this pull request. 

A similar check can be added to the DNSStub-Listener code ...
